examples/nat_topo.py

```diff
@@ -230,11 +237,15 @@ def sendInt(net, name, **args):
     # Set up the parser
     parser = argparse.ArgumentParser(description="Test the Pax NAT implementation.")
     parser.add_argument("action", choices=["run", "test"], nargs="?", default="run")
-    parser.add_argument("--no-X", help="don't launch additional windows'", action="store_false", dest="X_windows")
+    parser.add_argument("--no-X", help="don't launch additional windows", action="store_false", dest="X_windows")
+    parser.add_argument("--hold-open", help="leave xterm windows open", action="store_true", dest="hold_open")
+    parser.add_argument("--cli-first", help="provide cli access before running the tests", action="store_true", dest="cli_first")
```

Comment:
I'm not sure I understand what this option provides -- is the user given access to the CLI and it's up to them to start the tests from there, or are the tests started automatically after some interval (after the user has been given the CLI)?